src/SFA.DAS.EmployerFinance.MessageHandlers/EventHandlers/EmployerFinance/StartedProcessingLevyDeclarationsAdHocEventHandler.cs

```diff
@@ -0,0 +1,26 @@
+using System.Threading.Tasks;
+using MediatR;
+using NServiceBus;
+using SFA.DAS.EmployerFinance.Application.Commands.ImportLevyDeclarations;
+using SFA.DAS.EmployerFinance.Application.Commands.UpdateLevyDeclarationSagaProgress;
+using SFA.DAS.EmployerFinance.Messages.Events;
+
+namespace SFA.DAS.EmployerFinance.MessageHandlers.EventHandlers.EmployerFinance
+{
+    public class StartedProcessingLevyDeclarationsAdHocEventHandler : IHandleMessages<StartedProcessingLevyDeclarationsAdHocEvent>
```

Comment:
We could use a common generic base class for this and `StartedProcessingLevyDeclarationsEventHandler`, not so much for code sharing , but more from a self-documenting code benefit, i.e. to show that they do the same thing.

Reply:
They do slightly different things now 😉